llama_stack/providers/remote/inference/ollama/ollama.py

```diff
@@ -180,7 +180,6 @@ async def _generate_and_convert_to_openai_compat():
     async def _nonstream_completion(self, request: CompletionRequest) -> AsyncGenerator:
         params = await self._get_params(request)
         r = await self.client.generate(**params)
-        assert isinstance(r, dict)
```

Comment:
@wukaixingxp are you sure this does not happen when you don't use Llama Stack as a library (i.e., via server)? It seems like it should always fail.

Reply:
We never encounter this during our RAG app testing before.. not sure why..

Reply:
@wukaixingxp so this is a Completion request (not a ChatCompletion request) -- maybe you don't use this API